### PR TITLE
feat(update): suggest installing browsers on update

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/installer.ts [hunk 1]
```diff
@@ -86,3 +87,62 @@ export async function installPlaywright(vscode: vscodeTypes.VSCode) {
...
+    description: '— powers Mozilla Firefox',
+  };
+  const webkit: vscodeTypes.QuickPickItem = {
+    label: 'WebKit',
+    picked: true,
+    description: '— powers  Apple Safari',
+  };
+  const installDepsAction: vscodeTypes.QuickPickItem = {
+    label: 'Install Linux dependencies',
+    picked: process.platform === 'linux' && !fs.existsSync(path.join(process.env.XDG_CACHE_HOME || path.join(os.homedir(), '.cache'), 'ms-playwright')),
```
Comment (yury-s) [inline src/installer.ts:109]: why do we need this `!fs.existsSync(path.join(process.env.XDG_CACHE_HOME` ?

File: src/extension.ts [hunk 2]
```diff
@@ -582,7 +594,9 @@ located next to Run / Debug Tests toolbar buttons.`);
   }
 
   private _testMessageForTestError(testItem: vscodeTypes.TestItem, error: TestError): vscodeTypes.TestMessage {
-    const text = error.stack || error.message || error.value!;
+    let text = error.stack || error.message || error.value!;
+    if (text.includes('Looks like Playwright Test or Playwright'))
+      text = `Browser was not installed. Invoke 'Install Playwright Browsers' action to install missing browsers.`;
```
Comment (yury-s) [inline src/extension.ts:599]: will `Install Playwright Browsers` actually match the action ?

**Review by yury-s — APPROVED** [review]

File: src/installer.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/installer.ts]: yes!

File: src/extension.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/extension.ts:599]: Yes

File: src/installer.ts [hunk 3]
```diff
@@ -73,16 +52,87 @@ export async function installPlaywright(vscode: vscodeTypes.VSCode) {
...
+    args.push('chromium');
+  if (result.includes(firefoxItem))
+    args.push('firefox');
+  if (result.includes(webkitItem))
+    args.push('webkit');
+
+  if (args.length)
+    terminal.sendText(`npx playwright ${installCommand} ${args.join(' ')}`, true);
+  else if (result.includes(installDepsItem))
+    terminal.sendText(`npx playwright install-deps`, true);
```
Comment (mxschmitt) [inline src/installer.ts:108]: Sudo?

Comment (pavelfeldman) [reply src/installer.ts:108]: We don't do that currently.

**Pull request merged**